v2/scripts/generate_addresses_types.ts

```
@@ -0,0 +1,46 @@
+  return [...new Set(allValues)];
+};
+
+const generateTypesForKeys = (keys: string[]) => {
+  const networks = [...mainnetData, ...testnetData];
+
+  let typeDefs = "";
+
+  keys.forEach((key) => {
+    const uniqueValues = extractUniqueValues(networks, key);
+    if (uniqueValues.length > 0) {
+      const isNumeric = uniqueValues.every((value) => !isNaN(Number(value)));
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)